internal/pkg/template/diff/override.go

```diff
@@ -196,21 +196,20 @@ func (converter *getAttConverter) parse(from, to *yaml.Node, key string, overrid
 		toValue = to.Content[1]
 	}
 	// If the input node are of the same type (i.e. both seq or both scalar), parse them normally.
-	if fromValue.Kind == toValue.Kind {
-		return converter.intrinsicFuncMapTagConverter.parse(from, to, key, overrider)
-	}
 	// Otherwise, first convert the scalar input to seq input, then parse.
```

Comment:
nit: should we update the comment's "if...otherwise" accordingly?

Reply:
The logic is still the same, I just switched the `if` clause. 
```go
// If the input node are of the same type (i.e. both seq or both scalar), parse them normally.
// Otherwise, first convert the scalar input to seq input, then parse.
```
is still an accurate description of the code here.